lua/nvim_aider/init.lua

```diff
@@ -97,6 +97,16 @@ function M.setup(opts)
     end
   end, {})
 
+  vim.api.nvim_create_user_command("AiderQuickReadOnlyFile", function()
+    local absolute_path = utils.get_absolute_path()
+    if absolute_path == nil then
+      vim.notify("No valid file in current buffer", vim.log.levels.INFO)
+    else
+      local relative_filepath = vim.fn.fnamemodify(absolute_path, ":.")
```

Comment:
Any reason why using the relativ file path and not the absolut, like a few lines above?

Reply:
fixed